global.json

```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "3.1.102"
```

Comment:
What's the reason for this change? This seems to be a downgrade

Reply:
Jim mentioned on Slack it didn't work with 102 but only 101. Def something that would be interesting to know but I am not too worried about being one patch behind since PSSA is not a self-contained app

Reply:
it was a vsts build problem - for some reason the VSTS build machine couldn't download 102 - i didn't spend the time to find out why since it was just a single patch (for the SDK)